packages/mds-geography-service/service/validators.ts

```diff
@@ -26,18 +27,26 @@ import {
 } from '../@types'
 
 const ajvWithGeoJSON = new Ajv({ allErrors: true }).addKeyword({
-  keyword: 'valid-geojson',
+  keyword: 'valid_geojson',
   schema: false,
-  validate: (geography_json: unknown) => {
+  type: 'object',
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  validate: (geography_json: any, dataCtx: any) => {
```

Comment:
What is this `dataCtx` thing and if we're not using it in this function, can we rename to `_`?

Reply:
Might even be able to just remove it entirely (but the compiler might not like only having 1 arg instead of 2, idk)

Reply:
axing it was fine, thanks!